package.json

```diff
@@ -41,9 +41,7 @@
     "test-onlynode": "mocha --compilers js:babel/register test/node",
     "test-node": "npm run test-common && npm run test-onlynode",
     "test-browser": "testem",
-    "pretest-browser": "browserify test/browser/index.js -t babelify -o test/browser/page/tests-bundle.js",
-    "posttest-browser": "rm test/browser/page/tests-bundle.js",
-    "test": "npm run eslint && npm run test-node && npm run test-browser",
+    "test": "npm run eslint && npm run test-node && npm run test-browser -- ci -l PhantomJS",
```

Comment:
We should do `npm run test-browser -- ci -l PhantomJS` on the `travis-test` script, not `test` script. `test` script should stay as it was.